src/Actions/actions.js

```diff
@@ -25,10 +25,10 @@ function toggleModal(type, toggle){
     }
 }
 
-function updateUserLocation(zipcode){
+function updateCity(location){
     return {
-        type: 'UPDATE_USER_LOCATION',
-        data: zipcode,
+        type: 'UPDATE_CITY',
```

Comment:
what's the reasoning for changing location to city?

Reply:
feelings (JK!!! HAHAHAH, actually i dont know, city just rings better in my head 💯